packages/url/src/test/index.native.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Would it work just as well if this whole file was with:

```js
import './index.js';
```

?

Reply:
Good call, and it works! 🎉 
Best part is all tests are ✅ 

https://github.com/WordPress/gutenberg/pull/20172/commits/fa15b578c2598f2d4b4a02821e22bf32cc3932e6

Reply:
Oh nice we share tests with web, ignore my comment :)